src/SelectionSort.java

```diff
@@ -0,0 +1,38 @@
+public class SelectionSort {
+
+    public static void main(String[] args) {
+        int [] ints = {5, 7, 1, 3, 2, 0};
+
+        System.out.print("original array:\t");
+        print(ints);
+        System.out.println();
+        sort(ints);
+        System.out.print("sorted array:\t");
+        print(ints);
+    }
+
+    private static void sort(int [] ints) {
+        for (int i = 0; i < ints.length - 1; i++) {
+            int min = ints[i];
+            int minIndex = i;
+            for (int j = i; j < ints.length; j++) {
```

Comment:
better start second loop with `j = i + 1` to avoid unnecessary comparison `min = ints[j]' in the first iteration

Reply:
done